lib/elixir/test/elixir/fixtures/calendar/holocene.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Calendar.Holocene do
```

Comment:
would it make sense to add `@behaviour Calendar` and `@impl true` for this one? This way it's a more complete implementation for people to look at and base off of. Also, would probably be good to use `@impl true` for `Calendar.ISO`.

If any of this sounds good I'm happy to prepare PR, either against this branch or after this lands.